test/test_conservative_regridding.jl

```diff
@@ -0,0 +1,61 @@
+include("dependencies_for_runtests.jl")
+
+using ConservativeRegridding
+
+@testset "ConservativeRegridding extension" begin
+    @info "Testing ConservativeRegridding extension..."
+
+    # Test with LatitudeLongitudeGrid
+    @info "  Testing LatitudeLongitudeGrid regridding..."
+    coarse_grid = LatitudeLongitudeGrid(size=(90, 45, 1),
+                                        longitude=(0, 360),
+                                        latitude=(-90, 90),
+                                        z=(0, 1))
+
+    fine_grid = LatitudeLongitudeGrid(size=(180, 90, 1),
+                                      longitude=(0, 360),
+                                      latitude=(-90, 90),
+                                      z=(0, 1))
```

Comment:
Should we include a case with TripolarGrid?

Reply:
It is worth adding a test, although it mah need to be marked as broke  due to 

https://github.com/JuliaGeo/ConservativeRegridding.jl/issues/22